pandas/tests/indexing/test_iloc.py

```diff
@@ -462,71 +462,6 @@ def test_iloc_setitem_dups(self):
         df.iloc[[1, 0], [0, 1]] = df.iloc[[1, 0], [0, 1]].reset_index(drop=True)
         tm.assert_frame_equal(df, expected)
 
-    def test_iloc_getitem_frame(self):
```

Comment:
Do we still have coverage for this? Looks like main intention was .iloc testing so ix was secondary

Reply:
ill have to do a readthrough to check.  its not entirely easy to tell what kind of coverage we have for indexing